crates/nu-command/src/strings/str_/replace.rs

```diff
@@ -37,6 +37,10 @@ impl Command for SubCommand {
             .input_output_types(vec![
                 (Type::String, Type::String),
                 (Type::Table(vec![]), Type::Table(vec![])),
+                (
+                    Type::List(Box::new(Type::String)),
+                    Type::List(Box::new(Type::String)),
+                ),
```

Comment:
Mhh maybe not and there is a bug in that.

Reply:
I'm not following your meaning. Are you saying that `str replace` shouldn't take a list and return a list?

Reply:
I saw that you had to add the list types despite `.vectorizes_over_list()` so it seems like it doesn't so what it says.

Reply:
ok, ya. i'm not sure what the purpose to vectorizes_over_lists is then. i tried it in your previous comment and it didn't seem to do anything. maybe i'm just misunderstanding what it's purpose is.